graphql_compiler/tests/test_input_data.py

```diff
@@ -2344,11 +2344,37 @@ def filter_within_fold_scope() -> CommonTestData:  # noqa: D103
         Animal {
             name @output(out_name: "name")
             out_Animal_ParentOf @fold {
-                name @filter(op_name: "=", value: ["$desired"]) @output(out_name: "child_list")
-                description @output(out_name: "child_descriptions")
+                name @filter(op_name: "has_substring", value: ["$desired"])
+                     @output(out_name: "child_list")
             }
         }
     }"""
+    expected_output_metadata = {
+        "name": OutputMetadata(type=GraphQLString, optional=False, folded=False),
+        "child_list": OutputMetadata(type=GraphQLList(GraphQLString), optional=False, folded=True),
+    }
+    expected_input_metadata = {
+        "desired": GraphQLString,
+    }
+
+    return CommonTestData(
+        graphql_input=graphql_input,
+        expected_output_metadata=expected_output_metadata,
+        expected_input_metadata=expected_input_metadata,
+        type_equivalence_hints=None,
+    )
+
+
+def filter_and_multiple_outputs_within_fold_scope() -> CommonTestData:  # noqa: D103
+    graphql_input = """{
+            Animal {
+                name @output(out_name: "name")
+                out_Animal_ParentOf @fold {
+                    name @filter(op_name: "=", value: ["$desired"]) @output(out_name: "child_list")
+                    description @output(out_name: "child_descriptions")
+                }
+            }
+        }"""
```

Comment:
same here, seems like there is one extra step of indentation compared to equivalent code in surrounding test cases